src/main/java/io/debezium/connector/vitess/VitessErrorHandler.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.vitess;
+
+import io.debezium.connector.base.ChangeEventQueue;
+import io.debezium.pipeline.ErrorHandler;
+import io.grpc.StatusRuntimeException;
+
+public class VitessErrorHandler extends ErrorHandler {
+    public VitessErrorHandler(String logicalName, ChangeEventQueue<?> queue) {
+        super(VitessConnector.class, logicalName, queue);
+    }
+
+    @Override
+    protected boolean isRetriable(Throwable throwable) {
+        if (throwable instanceof StatusRuntimeException) {
+            final StatusRuntimeException exception = (StatusRuntimeException) throwable;
+            switch (exception.getStatus().getCode()) {
+                case UNAVAILABLE:
+                    return true;
+            }
```

Comment:
nit: Are we expecting to add any new error codes soon? I'm asking because we might be able to just use something like:

```suggestion
            return exception.getStatus().getCode().equals(UNAVAILABLE);
```